docs/commandline/pouch_kill.md

```diff
@@ -0,0 +1,49 @@
+## pouch kill
```

Comment:
Please remove this file. This file is auto-generated. https://github.com/alibaba/pouch/tree/master/docs#commandline
```
What's more, all commandline docs are auto generated via source code
```